### PR TITLE
Export EXECUTABLE-FIND and use it to dynamically choose between xclip and xsel

### PR DESCRIPTION
**Pull request by Ambrevar**
Since EXECUTABLE-FIND is rather useful, I thought we could export it.  It's a little bit out of scope of this package, but hey, rather more useful than less.

### CODE REVIEW

File: src/package.lisp [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 (defpackage trivial-clipboard
   (:use :cl #+os-windows :cffi)
-  (:export :text))
+  (:export :text
+           :executable-find))
```
Comment (snmsts) [inline src/package.lisp]: why should it be exported?

**Comment by Ambrevar** [comment]
As I said in the comment above, it does not have to be, but since it's rather useful and general enough, why not?

**Comment by snmsts** [comment]
Sorry to miss it...
because I don't want.

**Comment by Ambrevar** [comment]
OK, I've updated the pull request.

File: src/text.lisp [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+  (or (executable-find "xclip")
+      (executable-find "xsel")))
 
 (defvar *clipboard-out-command*
   #+(or darwin macosx)
   "pbpaste"
   #+(and :unix (:not :darwin))
-  "xclip")
+  (or (executable-find "xclip")
+      (executable-find "xsel")))
```
Comment (snmsts) [inline src/text.lisp:24]: if ``xclip`` isn't installed ``executable-find`` will be called fourtimes. can it be ``*clipboard-in-command*`` instead?

**Comment by Ambrevar** [comment]
You are right, I've fixed this.

**Comment by Ambrevar** [comment]
Thanks!

**Comment by snmsts** [comment]
Thank you!

**Pull request merged**